source/FileCache.cs

```diff
@@ -75,38 +75,31 @@ public static async Task<MemoryStream> HitAsync(string url)
                 memoryStream.Seek(0, SeekOrigin.Begin);
                 return memoryStream;
             }
+            var client = new HttpClient();
```

Comment:
thank you for helping to move this to the modern `HttpClient`, from the current `WebRequest` implementation! 🎉